src/directives/modal.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileoverview Provides the "ngeoModal" directive. This directive allows
+ * displaying a modal window. The directive takes care or the show/hide
+ * actions.
+ *
+ *
+ * Example:
+ *
+ * <modal shown="modalShown">
```

Comment:
Even:

`<modal shown="modalShown">`  -> `<ngeo-modal ngeo-modal-shown="modalShown">`. This is to match our conventions.